### ISSUE TITLE
Devil May Cry weapons cause significant FPS drops when being held

### ISSUE THREAD

**Issue by Drakyl** [kreezxil/Bane-of-Dragons#6]
**Describe the bug**
When Holding a Devil May Cry weapon FPS drops. switching to something else clears it.  The weapons are nice to look at, but seem to have negative effects on users FPS even at low graphic settings.

**To Reproduce**
1. Hold Devil May Cry weapon in main hand

**Comment by kreezxil** [comment]
Make sure AzureZhen gets a copy of this report

**Comment by Drakyl** [comment]
Reported on mods github as well

**Comment by Drakyl** [comment]
got response that it is a known issue and he's working on it

**Comment by AzureDoom** [comment]
In case this pack ever resurfaces, this issue has been fixed.

**Comment by kreezxil** [comment]
@AzureDoom did you just now fix it?

**Comment by AzureDoom** [comment]
Yeah, I finally took a crack at the DMC mod again and figured out how to make the models less complex, getting rid of the FPS drops.

**Comment by kreezxil** [comment]
Nice, considering I've added your mod to quite a few of my packs since you
first asked me if I would, I've got some updating to do. How far back did
you go with your fixes?

**Comment by AzureDoom** [comment]
1.12.2 and 1.15.2 have the new and improved models. 1.15.2 is more updated but 1.12.2 got the fixes too.

**Comment by kreezxil** [comment]
ok, so don't even touch the 1.14.4 yet. when you get the 1.14.4 done, ping
me in admin channel on the discord and then I'll proceed to update my pack
assets from 1.12.2+

**Comment by AzureDoom** [comment]
Unless the model system (https://github.com/MinecraftForge/MinecraftForge/pull/6180) is merged into Forge 1.14.4, 1.14.4 won't be seeing an update.

**Comment by kreezxil** [comment]
ok.
